dd-sdk-android/src/main/kotlin/com/datadog/android/rum/TrackingStrategy.kt

```diff
@@ -0,0 +1,23 @@
+package com.datadog.android.rum
+
+import android.app.Activity
+
+/**
+ * The TrackingStrategy interface.
+ */
+interface TrackingStrategy {
```

Comment:
You're making the assumption that a Tracking strategy is necessarily linked with an Activity lifecycle. I'd prefer to have a `register(context : Context)` and `unregister(context: Context)` instead.

Reply:
You can if you want create an ActivityLifecycleAwareTrackingStrategy abstract class to add the lifecycle callbacks and do something on activity start/stop, but i'd prefer the main TrackingStrategy to be as generic as possible.

Reply:
Yeah will change back and will try a different approach...I think I have the right solution now ;). Tks